src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue

```diff
@@ -2,6 +2,7 @@
   <div>
     <upload-header :document-type="documentType" />
     <drop-area
+      v-if="!isSelfie"
```

Comment:
no need to check selfie it or not, just remove drop-area will be enough

Reply:
done